src/site/content/en/blog/media-session/index.md

```diff
@@ -390,6 +400,79 @@ Resetting the position state is as easy as setting it to `null`.
 navigator.mediaSession.setPositionState(null);
 ```
 
+## Video conferencing actions {: #video-conferencing-actions }
+
+When the user puts their video call into a Picture-in-Picture window, the
+browser may display controls for mute/unmute, turnon/turnoff camera, and hang
```

Comment:
ditto about mute/unmute microphone

Reply:
This is on the verge of getting wordier than it needs to be to convey the idea (Distinguishing between two types of muting/unmuting will make the whole like clunky). I would go with something like:

>...the browser may display controls the mic, camera, and speaker and for hanging up.

Reply:
I had some typos in that comment, which I've now fixed.

Reply:
Done.
I've removed `speaker` as it's not supported.

Reply:
LGTM.